src/main/resources/avro/bdg.avdl

```diff
@@ -57,8 +57,55 @@ record Contig {
   union { null, string } species = null;
 }
 
+record Sequence {
+  union { null, string } bases;
+  union { null, string } qualities;
+}
+
+/**
+   The DNA fragment that is was targeted by the sequencer, resulting in
+   one or more reads.
+*/
+record Fragment {
+  /**
+     The name of this Fragment.
+  */
+  union { null, string } readName = null;
+
+  union { null, string } instrument;
+	union { null, string } runId;
+  /**
+     The following are Illumina specific, is it a problem?
+   */
+  union { null, string } flowcellId;
+  union { null, int} lane = null;
+  union { null, int} tile = null;
+  union { null, int} surface = null;
+  union { null, int} xPosition = null;
+  union { null, int} yPosition = null;
+
+  /**
+    Fragment's insert size derived from alignment, if the reads have been
+    aligned.
```

Comment:
Or sometimes there's a fragment size distribution, right?  How much detail do we want?